Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -7,10 +7,27 @@
 #include "Core/PowerPC/PowerPC.h"
 #include "Core/PowerPC/JitArm64/Jit.h"
 #include "Core/PowerPC/JitArm64/JitAsm.h"
+#include "Core/PowerPC/JitCommon/JitAsmCommon.h"
 #include "Core/PowerPC/JitCommon/JitCache.h"
 
 using namespace Arm64Gen;
 
+static void WriteDual32(u64 value, u32 address)
+{
+	Memory::Write_U32(value, address);
+	Memory::Write_U32(value >> 32, address + 4);
+}
+static void WriteDual16(u32 value, u32 address)
+{
+	Memory::Write_U16(value, address);
+	Memory::Write_U16(value >> 16, address + 2);
+}
+static void WriteDual8(u16 value, u32 address)
+{
+	Memory::Write_U8(value, address);
+	Memory::Write_U8(value >> 8, address + 1);
```

Comment:
It would be more conventional to write this as `Memory::Write_U16(Common::swap16(value), address);`